config/puma/production.rb

```diff
@@ -1,3 +1,11 @@
+pid_dir = '/var/run/foreman/pids'
+
+# Store the pid of the server.
+pidfile "#{pid_dir}/puma.pid"
```

Comment:
Reading further about this, the default pid is `tmp/pids/server.pid` and I don't see the need to change this. I also think that `rails server` has a `--pid` option to change this and there's no real reason to hardcode it. In systemd we don't even care about pids at all since it's all managed for us. Am I missing something?

Reply:
you are right, we are concerned about the `state_path` option only and there is no need to change `pidfile` option. I changed it to make sure the `pid` and `state` files live together, but it doesn't matter. I will remove this entry.